html/browser.html

```diff
@@ -20,30 +20,79 @@
     <script id="mineral-observation-template" type="text/x-handlebars-template">
         <tr>
             <td>
-                {{#if mineral.mineral-name-plain}}<div><strong>Mineral Name:</strong> <a href="{{mineral.mineral-mindat-url}}" target="_blank"> {{mineral.mineral-name-plain}}</a></div>
-                {{else}} <div><strong>Mineral Name:</strong> N/A</div>
+                {{#if mineral-species.mineral-name-plain}}<div><strong>Mineral Name:</strong><a href="{{mineral-species.mineral-mindat-url}}" target="_blank">{{mineral-species.mineral-name-plain}}</a></div>
```

Comment:
I think we should add back the space between the semi-colon and the mineral name